groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -480,6 +480,17 @@ groups:
       - dlipovetsky@d2iq.com
       - neolit123@gmail.com
 
+  - email-id: k8s-infra-staging-kubespray@kubernetes.io
+    name: k8s-infra-staging-kubespray
+    description: |-
+      ACL for staging kubespray artifacts.
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - kay.yan@daocloud.io
+      - mg@max.gautier.name #VannTen
```

Comment:
```suggestion
      - mg@max.gautier.name
```

Reply:
Thanks @neolit123 

Thank you very much for your detailed guidance. It has been modified.